packages/appstore/src/components/wallet-action-button/wallet-action-button.scss

```diff
@@ -0,0 +1,23 @@
+.wallet-action-button {
+    background-color: #fff;
+    border-radius: 16px;
+    cursor: pointer;
+    align-items: center;
+    display: flex;
+    border: 1px solid #999999;
```

Comment:
Can we use `--button-secondary-default` instead?

Reply:
done :heavy_check_mark: